### PR TITLE
AESinkAudiotrack: Use 192 khz for TrueHD / DTS-HD RAW as well

### PR DESCRIPTION
**Pull request by fritsch**
The 48 khz is historically chosen. There is no way, that this bandwidth would be enough to do DTS-HD-MA or TrueHD. With this new check, we would not list that format anymore - as in the past - it auto fellback to normal DTS and on some machines failed for TrueHD directly.

See also: https://forum.kodi.tv/showthread.php?tid=378616

### CODE REVIEW

**Comment by thexai** [comment]
Tested on Shield and TrueHD RAW continues working same as before. It's great if this change enables it on more devices.

Although in the case of the forum user it seems that TrueHD is converted to PCM by the TV (soundbar receives PCM) but at least it works.

**Comment by fritsch** [comment]
I would keep 48 khz for DTS-HD-MA to not break FireTV's RAW DTS-HD-MA even if their packer would reduce it to DTS-Core - and only change it for TrueHD. Do you agree with my latest version as well?

File: xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp [hunk 1]
```diff
@@ -1073,6 +1080,7 @@ void CAESinkAUDIOTRACK::UpdateAvailablePassthroughCapabilities(bool isRaw)
   m_info.m_wantsIECPassthrough = false;
   m_info.m_dataFormats.push_back(AE_FMT_RAW);
   m_info.m_streamTypes.clear();
+  bool supports_192khz = m_sink_sampleRates.find(192000) != m_sink_sampleRates.end();
```
Comment (thexai) [inline xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp:1083]:
```suggestion
  const bool supports_192khz = m_sink_sampleRates.find(192000) != m_sink_sampleRates.end();
```

And maybe add blank line after

Comment (fritsch) [reply xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp:1083]: Thanks - added it.

**Review by thexai — APPROVED** [review]

**Pull request merged**